pkg/kubectl/cmd/certificates.go

```diff
@@ -82,10 +82,11 @@ func (o *CertificateOptions) Complete(f cmdutil.Factory, cmd *cobra.Command, arg
 	}
 
 	o.builder = f.NewBuilder()
-	o.clientSet, err = f.KubernetesClientSet()
+	clientSet, err := f.KubernetesClientSet()
```

Comment:
You should get config from factory and then instantiate the client manually, see https://github.com/kubernetes/kubernetes/blob/0e31372b2fa6dc9e6a32f4be09c088ec0adfdf8d/pkg/kubectl/cmd/create/create_job.go#L115-L122